pkg/hubble/metrics/api/context.go

```diff
@@ -214,17 +214,14 @@ func (ls labelsSet) HasLabel(label string) bool {
 }
 
 func (ls labelsSet) String() string {
-	var b strings.Builder
+	var labels []string
```

Comment:
The commit says "Fix", but what is it fixing and why should be described in the commit msg.